src/deletepy/utils/logging_utils.py

```
@@ -0,0 +1,276 @@
+    
+    return setup_logging(
+        level=level,
+        log_file=log_file,
+        structured=structured,
+        operation=operation
+    )
+
+
+class LogContext:
+    """Context manager for adding structured context to log records."""
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)